### PR TITLE
[OSDOCS-11541]: Security and compliance- Classic-HCP migration

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.16+
Issue:

https://issues.redhat.com/browse/OSDOCS-11541

Link to docs preview:

https://80286--ocpdocs-pr.netlify.app/openshift-rosa/latest/security/audit-log-view
QE review:
- [ ] QE has approved this change.

Additional information:

HCP Book Content Port

### CODE REVIEW

**Comment by mletalie** [comment]
SME review complete

**Comment by mletalie** [comment]
Hello @xueli181114, 
   QE review considerations (based on what I have been told by PM and MOBB SME):  
1) The topic "rosa-adding-additional-constraints-for-ip-based-aws-role-assumption" is  ROSA flavor-agnostic; that is, there is no change between how this is applied to Classic vs HCP, so it can stay the same.
2) The topic "audit-log-view" is not applicable to ROSA HCP, so it will not be included in those docs. It will stay the same for ROSA Classic.

Given that, this will hopefully be one of the easier doc migrations to QE review. Thanks!

**Comment by yingzhanredhat** [comment]
LGTM

File: security/audit-log-view.adoc [hunk 1]
```diff
@@ -33,4 +34,4 @@ ifndef::openshift-rosa,openshift-dedicated[]
 * xref:../security/audit-log-policy-config.adoc#audit-log-policy-config[Configuring the audit log policy]
 endif::[]
 * xref:../observability/logging/log_collection_forwarding/log-forwarding.adoc#log-forwarding[About log forwarding]
-
+endif::[]
```
Comment (EricPonvelle) [inline security/audit-log-view.adoc:37]: I don't think you need to include the closing conditional distro, but it helps when multiple conditionals are going on.

```suggestion
endif::openshift-rosa-hcp[]
```

**Review by EricPonvelle — COMMENTED** [review]
Small suggestion, but it looks good to me otherwise.

File: security/audit-log-view.adoc (hunk 1 above)
Comment (mletalie) [reply security/audit-log-view.adoc:37]: ah, good point....thx!

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Pull request merged**